$DragonFly: src/lib/libpam/patches/openpam_ttyconv.c.patch,v 1.1 2005/07/13 12:34:21 joerg Exp $

--- openpam_ttyconv.c.orig	2005-07-07 19:46:23.000000000 +0200
+++ openpam_ttyconv.c	2005-07-07 19:46:26.000000000 +0200
@@ -65,7 +65,7 @@
 	char buf[PAM_MAX_RESP_SIZE];
 	struct sigaction action, saved_action;
 	sigset_t saved_sigset, sigset;
-	unsigned int saved_alarm;
+	unsigned int saved_alarm = 0;
 	int eof, error, fd;
 	size_t len;
 	char *retval;
